frontend/public/components/masthead-toolbar.jsx

```diff
@@ -53,6 +53,8 @@ import { FeedbackModal } from '@patternfly/react-user-feedback';
 import { useFeedbackLocal } from './feedback-local';
 import feedbackImage from '@patternfly/react-user-feedback/dist/esm/images/rh_feedback.svg';
 
+const LAST_CONSOLE_ACTIVITY_TIMESTAMP_LOCAL_STORAGE_KEY = 'last-console-activity-timestamp';
```

Comment:
just a though to align with the actual feature:
```suggestion
const LAST_INACTIVITY_TIMEOUT_RESET_TIMESTAMP_LOCAL_STORAGE_KEY = 'last-inactivity-timeout-reset-timestamp';
```

Reply:
👍 Naming is hard.